components/card/demo/no-padding.md

```diff
@@ -1,10 +1,18 @@
 ---
 order: 3
-title: 更灵活的内容展示
+title:
+  zh-CN: 更灵活的内容展示
+  en-US: More flexible content
 ---
 
+## zh-CN
+
 可以调整默认边距，设定宽度。
 
+## en-US
+
+An advanced usage of a card, adjusting the width and the default margins.
```

Comment:
I think we are trying to say that setting default width and margin is itself an advanced usage, so "Advanced usage of customizing" might be a bit wrong here. Perhaps, "Customizing advanced Card properties like default width and margin?" or remove advanced altogether, "Customizing default width and margin".